### PR TITLE
Replaced SIGKILL with SIGINT

### PR DESCRIPTION
**Pull request by aandryashin**

### CODE REVIEW

File: service/driver.go [hunk 1]
```diff
@@ -70,7 +71,7 @@ func (d *Driver) StartWithCancel() (*StartedService, error) {
 
 func (d *Driver) stopProcess(cmd *exec.Cmd) {
 	log.Printf("[%d] [TERMINATING_PROCESS] [%d]\n", d.RequestId, cmd.Process.Pid)
-	err := cmd.Process.Kill()
+	err := cmd.Process.Signal(syscall.SIGINT)
```
Comment (vania-pooh) [inline service/driver.go]: The crossplatform signal value is: `os.Interrupt`.

Comment (vania-pooh) [reply service/driver.go]: Also commit message has a typo.

**Comment by aandryashin** [comment]
Signal on windows is not implemented.

**Comment by sskorol** [comment]
@aandryashin @vania-pooh if this PR was about killing ffmpeg process, I've researched this question for Windows before. There's a command line tool written on C ages ago, which emulates CTRL+C key press behavior for specified process. It allows to gracefully stop ffmpeg process (actually this is the only valid way on Windows I believe). Not sure if it could be easily emulated on Go. 

Anyway you can download it from my [repo](https://github.com/sskorol/docker-selenium-grid/blob/master/src/main/resources/SendSignalCtrlC.exe).

And here's a usage [example](https://github.com/sskorol/docker-selenium-grid/blob/master/src/main/java/com/blogspot/notes/automation/qa/grid/utils/VideoRecordingUtils.java#L59-L64) on java.

**Comment by vania-pooh** [comment]
@sskorol no, this is about killing webdriver process when working without Docker on Windows.

**Pull request closed without merging**